### DISCUSSION TITLE
TPI Spec Discussion

### DISCUSSION THREAD

**Discussion by smartopian** [KantaraInitiative/ancr-wg#13] [General]
I have been working on the TPI Scheme and I realised i have it wrong in v0.6.  

The TPI's are compliance assessments, which use an international standard format and framework to demonstrate evidentiary conformance.   

So in this regard, the TPI's are a compliance assessment, using conformant records.  Needless to say the scheme structure is good but not correct.

**Comment by IDmachines** [comment]
Yes.

 

Suggest code of conduct and consensus protocol are inputs (profiles?, e.g. Quebec/PCTF/CoE108+) into evidentiary conformance. 

 

Suggest the TPIs measure compliance of the controller credential, including dynamic aspects, in context. It then depends on the type of controller (which assumes you have cleared the TPI gateway) credential. We need to work up to the Consent Notice Receipt and Controller (and then AuthC and tokens). The TPIs are the start.

 

Suggest a next step roadmap is the Notice Controller Credential.

 

From: Mark Lizar ***@***.***> 
Sent: Thursday, August 17, 2023 12:36 AM
To: KantaraInitiative/WG-ANCR ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [KantaraInitiative/WG-ANCR] TPI Spec Discussion (Discussion #13)

 

I have been working on the TPI Scheme and I realised i have it wrong in v0.6.

The TPI's are compliance assessments, which use an international standard format and framework to demonstrate evidentiary conformance.

So in this regard, the TPI's are a compliance assessment, using conformant records. Needless to say the scheme structure is good but not correct.
